frontend/src/components/auth/Register.js

```diff
@@ -41,7 +47,7 @@ class Register extends Component {
         const data = {
             email: this.state.email,
             password: this.state.password,
-            password_confirmation: this.state.passwordConfirmation
+            passwordConfirmation: this.state.passwordConfirmation
         };
 
         this.props.registerUser(data, this.props.history);
```

Comment:
maybe just `this.props.registerUser(...this.state, this.props.history)`

Reply:
right